packages/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -509,6 +509,48 @@ describe('CallingClient Tests', () => {
       expect(registerSpy).not.toBeCalled();
     });
 
+    it('Simulate a network flap before initial registration is done should not trigger re-verify registration: Restore Failure', async () => {
+      expect(line.getRegistrationStatus()).toEqual(MobiusStatus.ACTIVE);
+
+      reg.deregister();
+      line.registration.deregister();
```

Comment:
Isn't reg = line.registration ? Why are we called line.registration.deregister() after this?
Also, why are we not doing line.deregister() directly?

Reply:
Ref:[/CallingClient/line/index.ts#L148]  (https://github.com/webex/webex-js-sdk/blob/a57a4decebcf721da6495c398b566e9b803fb6c3/packages/calling/src/CallingClient/line/index.ts#L148)
 within the line Object we have `registration` module..
 
 [the way we are accessing the registration modules method] (https://github.com/webex/webex-js-sdk/blob/a57a4decebcf721da6495c398b566e9b803fb6c3/packages/calling/src/CallingClient/CallingClient.test.ts#L377)

Reply:
> Isn't reg = line.registration ? Why are we called line.registration.deregister() after this? Also, why are we not doing line.deregister() directly?

Ref:[/CallingClient/line/index.ts#L148] (https://github.com/webex/webex-js-sdk/blob/a57a4decebcf721da6495c398b566e9b803fb6c3/packages/calling/src/CallingClient/line/index.ts#L148C1-L150C4)
 within the line Object we have `registration` module..
 
 [the way we are accessing the registration modules method](https://github.com/webex/webex-js-sdk/blob/a57a4decebcf721da6495c398b566e9b803fb6c3/packages/calling/src/CallingClient/CallingClient.test.ts#L377)

Reply:
I still don't understand why we are doing it this way.
Why can't we just have 1 line saying `line.deregister()`?

Reply:
@karthikraji2020

> so to access the deregister() we are using line.registration.deregister()

here, `reg == line.registration` as Sreekanth mentioned above so no need to separately call `line.registration.deregister()` to invoke deregister(). `reg.deregister()` will invoke the same function.

Use either of below not both,
1. `reg.deregister()`
2. `line.registration.deregister()`

Reply:
ack: updated